### PR TITLE
Dockerfile: uninstall dev tools to shrink image

### PR DESCRIPTION
**Pull request by unclejack**
This PR modifies the Dockerfile to reduce the size of the image. The original size of the image was 220 MB. The new image is only 87.5 MB.

### CODE REVIEW

File: install/ansible/Dockerfile [hunk 1]
```diff
@@ -1,5 +1,8 @@
 FROM alpine:3.5
 MAINTAINER contiv
 
-RUN apk add --no-cache python-dev py-pip gcc musl-dev openssl-dev libffi-dev && \
-    pip install --upgrade pip && pip install ansible==2.2.0.0
+RUN DEV_PACKAGES="python-dev py-pip gcc musl-dev openssl-dev libffi-dev" \
+ && apk add --no-cache python openssl libffi $DEV_PACKAGES \
+ && pip install --upgrade pip \
+ && pip install ansible==2.2.0.0 \
```
Comment (unclejack) [inline install/ansible/Dockerfile]: @neelimamukiri: Do we really need 2.2.0.0? We should use 2.2.1.0 because it has security updates.

Comment (neelimamukiri) [inline install/ansible/Dockerfile]: @unclejack, is our ansible swarm setup works fine with 2.2.1.0, we can update this as well.

File: install/ansible/Dockerfile [hunk 2]
```diff
@@ -1,5 +1,8 @@
...
+ && apk del $DEV_PACKAGES
```
Comment (neelimamukiri) [inline install/ansible/Dockerfile]: Awesome! This looks great 👍

**Comment by unclejack** [comment]
I've updated the PR to install ansible 2.2.1.0.

**Pull request merged**